pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -393,6 +429,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: BasicUserRoleName,
+				Annotations: map[string]string{
+					constants.OpenShiftDescription: "A user that can get basic information about projects and users."},
```

Comment:
this allows listing projects and getting information about yourself, not users in general